README.md

```diff
@@ -1,241 +1,133 @@
-      "icon": "assets/osx/mount.icns",
-      "icon-size": 80,
-      "contents": [
-        { "x": 438, "y": 344, "type": "link", "path": "/Applications" },
-        { "x": 192, "y": 344, "type": "file" }
-      ]
-    },
-    "win" : {
-      "title" : "Loopline Systems",
-      "version" : "x.x.x.x",
-      "publisher": "Publisher Info",
-      "icon" : "assets/win/icon.ico",
-      "verbosity": 1,
-      "nsiTemplate" : "path/to/custom/installer.nsi.tpl",
-      "fileAssociation": {
-        "extension": ".loop",
-        "fileType": "Loopline Systems File"
-      }
-    },
-    "linux" : {
-      "arch" : 64,
-      "target" : "deb",
-      "version" : "x.x.x.x",
-      "title" : "Loopline Systems",
-      "comment" : "This is a comment",
-      "executable" : "myExec",
-      "maintainer": "Dummy Maintainer <dummy@maintainer.org>"
-    }
-  }
-}
-```
+As you can see, you need to customize OS X options only if you want to provide custom `x, y`.
+Don't customize paths to background and icon, — just follow conventions (if you don't want to use `build` as directory of resources — please create issue to ask ability to customize it).
 
-**Note:** Need to add something that might have value for others? Consider a [Pull Request](https://github.com/loopline-systems/electron-builder/pulls)! ;)
+See [OS X options](https://www.npmjs.com/package/appdmg#json-specification) and [Windows options](https://github.com/electronjs/windows-installer#configuring).
 
+# Auto Update
+`electron-builder` produces all required artifacts:
 
-## How we use it so far
+* `.dmg`: OS X installer, required for OS X user to initial install.
+* `-mac.zip`: required for Squirrel.Mac.
+* `.exe` and `-x64.exe`: Windows installer, required for Windows user to initial install. Please note — [your app must handle Squirrel.Windows events](https://github.com/electronjs/windows-installer#handling-squirrel-events). See [real example](https://github.com/develar/onshape-desktop-shell/blob/master/src/WinSquirrelStartupEventHandler.ts).
+* `.full-nupkg`: required for Squirrel.Windows.
+* `-amd64.deb` and `-i386.deb`: Linux Debian package. Please note — by default the most effective [xz](https://en.wikipedia.org/wiki/Xz) compression format used.
 
-When you run `npm run pack`, it will create executables for the Windows and OS X platforms inside of the `dist` directory. It grabs the generated executables afterwards to create the installers out of it.
+You need to deploy somewhere [releases/downloads server](https://github.com/GitbookIO/nuts).
 
+In general, there is a possibility to setup it as a service for all (it is boring to setup own if cloud service is possible).
+May be it will be soon (feel free to file an issue to track progress).
+It is safe since you should sign your app in any case (so, even if server will be compromised, users will not be affected because OS X will just block unsigned/unidentified app).
 
-directory structure
+# Code signing
```

Comment:
Thanks, it is already fixed in this PR, and fixed in the electron-complete-builder docs.